learntools/machine_learning/nbs/ch3.ipynb

```diff
@@ -121,6 +124,10 @@
    "source": [
     "# Create the list of predictors below\n",
     "# predictor_names = ___\n",
```

Comment:
I dislike initializing to [] or None, because it's harder to see from scanning the code that this requires user work (sometimes we intentionally create variables that are empty lists).